plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/LibraryFactory.java

```diff
@@ -27,6 +27,8 @@
 import org.eclipse.core.runtime.IConfigurationElement;
 import org.eclipse.core.runtime.InvalidRegistryObjectException;
 
+// todo AppEngineLibraries and LibraryFactory should be in the same package and then this doesn't
+// need to be public
```

Comment:
If we move them to the same package, we should rename `AppEngineLibraries` so that its name indicates a more general functionality just like `LibraryFactory`'s name, as they can be used for any kind of library defined in plugin.xml, not only for App Engine related libraries.